website/content/docs/integrations/consul-integration.mdx

```diff
@@ -38,58 +37,143 @@ service discovery to connect tasks to other services deployed across your
 cluster. Nomad integrates with Consul to provide service discovery and
 monitoring.
 
-To configure a job to register with service discovery, please see the
+To configure a job to register with service discovery, please refer to the
 [`service` job specification documentation][service].
 
 ## Dynamic Configuration
 
-Nomad's job specification includes a [`template` block](/nomad/docs/job-specification/template)
-that utilizes a Consul ecosystem tool called [Consul Template](https://github.com/hashicorp/consul-template). This mechanism creates a convenient way to ship configuration files
-that are populated from environment variables, Consul data, Vault secrets, or just
-general configurations within a Nomad task.
+Nomad's job specification includes a [`template` block][] that uses a Consul
+ecosystem tool called [Consul Template][]. This mechanism creates a convenient
+way to ship configuration files that are populated from environment variables,
+Consul data, Vault secrets, or just general configurations within a Nomad task.
+
+For more information on Nomad's template block and how it leverages Consul
+Template, please see the [`template` job specification documentation][].
+
+## Using Nomad Workload Identity with Consul
+
+Starting in Nomad 1.7, Nomad clients will use a task or service's [Workload
+Identity][] to authenticate to Consul and obtain a Consul token specific to the
+service or task.
+
+### Configuring Consul Authentication
+
+Create a configuration file for a Consul [JWT Auth Method][]. The `JWKSURL`
+field should point to the Nomad servers. The remaining fields are required to
+match those shown here.
+
+```json
+{
+  "JWKSURL": "https://nomad.example.com:4646/.well-known/jwks.json",
+  "JWTSupportedAlgs": ["RS256"],
+  "BoundAudiences": "consul.io",
+  "ClaimMappings": {
+    "nomad_namespace": "nomad_namespace",
+    "nomad_job_id": "nomad_job_id",
+    "nomad_task": "nomad_task",
+    "nomad_service": "nomad_service"
+  }
+}
+```
 
-For more information on Nomad's template block and how it leverages Consul Template,
-please see the [`template` job specification documentation](/nomad/docs/job-specification/template).
+Using that configuration file, you'll create two different Consul Auth Methods:
```

Comment:
That rule makes sense generally but I'm trying to say in these 3 paragraphs "this is what the next sections are going to show you", rather than "do this now" without having the instructions immediately handy until the following sections. Got any suggestion for rewording for that?

Reply:
Ahh I see. No, I think that's a good use as is.